app/javascript/components/ingredients/base-input-form.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <input
+    class="appearance-none block w-full bg-white text-gray-700 border border-gray-200 rounded py-3 px-4 mb-3 leading-tight focus:outline-none"
```

Comment:
Lo mismo, me llama la atención lo larga de esta línea y que el mono no reclame

Reply:
Me llama la atención tb, @agomezcampero faltará configurar algo acá?

Reply:
Vi este mensaje del mono: `Remaining comments which cannot be posted as a review comment to avoid GitHub Rate Limit`, será por eso que no los mostró? En local el linter me está funcionando bien y me parece que está todo configurado bien en el config.yml